server/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "dependencies": {
     "body-parser": "^1.18.3",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v3.5.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-update-enums",
```

Comment:
Reminder to self to change this once https://github.com/bravetechnologycoop/brave-alert-lib/pull/24 is merged.

Reply:
Done.